### PR TITLE
qutebrowser: update to 0.10.1

### PR DESCRIPTION
**Pull request by NoctuaNivalis**

### CODE REVIEW

File: srcpkgs/qutebrowser/INSTALL.msg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+To use the new QtWebEngine backend, install python3-PyQt5-webchannel and
+python3-PyQt5-webengine.
```
Comment (The-Compiler) [inline srcpkgs/qutebrowser/INSTALL.msg]: Doesn't python3-PyQt5-webengine depend on python3-PyQt5-webchannel anyways?

Comment (NoctuaNivalis) [inline srcpkgs/qutebrowser/INSTALL.msg]: No, which is why this message is important. Everyone can figure out in a second you'll probably need python3-PyQt5-webengine for a webengine backend, but you need a few minutes and some error messages to figure out you need webchannel as well.

Comment (The-Compiler) [inline srcpkgs/qutebrowser/INSTALL.msg]: I think this should be fixed in python3-PyQt5-webengine. I don't think QtWebEngine even builds without QtWebChannel, so the wrappers should depend on each other too.

Comment (NoctuaNivalis) [inline srcpkgs/qutebrowser/INSTALL.msg]: Seems like it. Let's ask @asergi, the maintainer of python-PyQt5, if there's a reason not to have this dependency.

Comment (asergi) [inline srcpkgs/qutebrowser/INSTALL.msg]: Fixed in 268f46fbf8c7e13bd7001a4a2f5776e2f0d425a3. Thank you.

**Pull request merged**